### ISSUE TITLE
Dynamically active extension on fb page rather than always active on another web pages

### ISSUE THREAD

**Issue by imbipulkumar** [aviary-apps/Pelican#19]
The Pelican icon in the following screenshot represents active extension on the facebook.
![color](https://user-images.githubusercontent.com/31007652/50232456-fb915080-03d6-11e9-9a95-e7574a3f5293.PNG)

The Pelican icon in the following screenshot represents inactive extension on other web pages than facebook.
![greyscl](https://user-images.githubusercontent.com/31007652/50232468-02b85e80-03d7-11e9-890b-ebb473269077.PNG)

**Comment by gauthamzz** [comment]
Yes this is a great idea. Do send a pr, if you finish working on this.
